pkg/plugins/gateway/util.go

```diff
@@ -50,30 +50,20 @@ func validateStreamOptions(requestID string, user utils.User, jsonMap map[string
 	return nil
 }
 
+var defaultRoutingStrategy, defaultRoutingStrategyEnabled = utils.CheckEnvExists(EnvRoutingAlgorithm)
```

Comment:
`CheckEnvExists` doesn't seem like the best name. Maybe rename it to `LookupEnv` or just use os.LookupEnv directly?
https://github.com/vllm-project/aibrix/blob/6c325828b22098ef17240c7fbd50d032ce93449c/pkg/utils/redis.go#L33-L38

@Jeffwan What do you think?

Reply:
Sure, rename it to LookupEnv.

Reply:
sounds good.